pages/providers/pinterest.mdx

```diff
@@ -16,16 +16,17 @@ Head to [Pinterest Developer Dashboard](https://developers.pinterest.com/apps/)
 
 ### Step 2 Copy the App ID and Secret
 Copy the App ID at "App id" and the Secret Key at "App secret key"
-
+[Copy App ID and Secret](https://github.com/egelhaus/postiz-docs/blob/cfd39114ef6d44da864717b23d704f6265cefa84/public/images/providers/pinterest/pinterest-002.png)
 
 ### Step 3 Configure Redirect URI
 Add your Redirect URI at "Redirect URIs" with the default URI "http://localhost:4200/integrations/social/pinterest
 [Setup of Redirect URIs](https://github.com/egelhaus/postiz-docs/blob/d11ea333ad8d70c5dee7ac0ad76283ded514219b/public/images/providers/pinterest/pinterest-001.png)
 
 </Steps>
 
-
 ```env
 YPINTEREST_CLIENT_ID=""

```

Comment:
I know it's not your mistake, any chance you can fix this env variable?

Reply:
Fixed.